app/generators/cinema_halls/generate_seats.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module CinemaHall
+  module Generators
+    class GenerateSeats
+      AVAILABLE_COLUMNS = ('A'..'J')
+
+      def generate_seats(capacity)
```

Comment:
it would be even better if you define `call` method and use the class as `CienemaHall::Generators::GenerateSeats.new.call(capacity)` and inside this `call` method call `generate_seats` located in `private` methods

Reply:
@kodowicz why it would be better? :)

Reply:
@kodowicz I'm not sure if I understand this `CienemaHall::Generators::GenerateSeats.new.call(capacity)`
Do you mean something like:


```
class CinemaHall::Generators::GenerateSeats.new.call(capacity)
def call
generate_seats
end

private
def generate_seats
#some code
end
```

Reply:
I thought about this way of writing classes because I thought there could be some other logic to generate seats that could be implemented in the future. Also in my opinion it is more readable if called class is not repeating its name like `GenerateSeats.generate_seats`. If Paulina decides to extend the class by several methods it would be again more readable to use `call` method and inside it needed private method.